paths_cli/compiling/plugins.py

```diff
@@ -1,12 +1,13 @@
 from paths_cli.compiling.core import InstanceCompilerPlugin
 from paths_cli.plugin_management import OPSPlugin
 
+
 class CategoryPlugin(OPSPlugin):
     """
     Category plugins only need to be made for top-level
     """
     def __init__(self, plugin_class, aliases=None, requires_ops=(1, 0),
-                 requires_cli=(0,4)):
+                 requires_cli=(0, 3)):
```

Comment:
this seems like an actual bugfix? should this be mentioned somewhere?

Reply:
I'd argue that it isn't a bugfix on the basis that it isn't in released code. However, I edited the main post of this PR to make mention of the change.